README.md

```diff
@@ -1,6 +1,44 @@
-# get-country
-[![Travis CI](https://travis-ci.org/busrapidohq/get-country.svg?branch=master)](https://travis-ci.org/busrapidohq/get-country) [![Codecov](https://img.shields.io/codecov/c/github/busrapidohq/get-country/master.svg)](https://codecov.io/gh/busrapidohq/get-country) [![npm](https://img.shields.io/npm/dm/get-country.svg)](https://www.npmjs.com/package/get-country) [![npm version](https://img.shields.io/npm/v/get-country.svg)](https://www.npmjs.com/package/get-country) [![npm dependencies](https://david-dm.org/busrapidohq/get-country.svg)](https://david-dm.org/busrapidohq/get-country) [![npm dev dependencies](https://david-dm.org/busrapidohq/get-country/dev-status.svg)](https://david-dm.org/busrapidohq/get-country#info=devDependencies)  
+# country-code
+[![Travis CI](https://travis-ci.org/busrapidohq/country-code.svg?branch=master)](https://travis-ci.org/busrapidohq/country-code) [![Codecov](https://img.shields.io/codecov/c/github/busrapidohq/country-code/master.svg)](https://codecov.io/gh/busrapidohq/country-code) [![npm](https://img.shields.io/npm/dm/country-code.svg)](https://www.npmjs.com/package/country-code) [![npm version](https://img.shields.io/npm/v/country-code.svg)](https://www.npmjs.com/package/country-code) [![npm dependencies](https://david-dm.org/busrapidohq/country-code.svg)](https://david-dm.org/busrapidohq/country-code) [![npm dev dependencies](https://david-dm.org/busrapidohq/country-code/dev-status.svg)](https://david-dm.org/busrapidohq/country-code#info=devDependencies)
 🗺 Get ISO 3166-1 alpha-3 country code for geographic coordinates.
 
-`test/fixtures/cities.json` was exctracted from data licensed under a Creative Commons Attribution 3.0 License,
-see [cc3](http://creativecommons.org/licenses/by/3.0/) from geonames.org
+## Install
+
+```bash
+$ npm install --save country-code
+$ npm install --save world-countries-boundaries-100m
+```
+## Countries bounds map file
+In order to use this package you need to provide a valid `GeoJSON` FeatureCollection object with the `ISO_A3` property set for each country.
+You can get one with customizable accuracy from [world-countries-boundaries](https://github.com/busrapidohq/world-countries-boundaries) repo.
+
+## Usage
+
+```javascript
+const countryCode = require('country-code');
+
+// Load a GeoJSON with country boundaries
+countryCode.use(require('world-countries-boundaries-1m')());
+
+// query a point
+countryCode.get(42.50779, 1.52109);
+// > 'AD'
+```
+
+## API
+### .use(geoJson)
+Load a geoJson and preprocess the data to speed up future queries
+- *geoJson*, Object, a GeoJson FeatureCollection. You can get one with customizable accuracy at https://github.com/busrapidohq/world-country-boundaries
+### .get(lat, lng)
```

Comment:
### get(lat, lng)

Searches for every country which contains the point (lat, lng).

#### lat

Type: `number`

Latitude of the point.

#### lng

Type: `number`

Longitude of the point.